src/configs/os.rs

```diff
@@ -0,0 +1,71 @@
+use indexmap::{indexmap, IndexMap};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(feature = "config-schema", derive(schemars::JsonSchema))]
+#[serde(default)]
+pub struct OSConfig<'a> {
+    pub format: &'a str,
+    pub style: &'a str,
+    pub symbols: IndexMap<String, &'a str>,
```

Comment:
This doesn't need to be owned.
```suggestion
    pub symbols: IndexMap<&'a str, &'a str>,
```

Reply:
I followed the solution that the `directory` module uses for its own `IndexMap` as the deserializer parses the config into an owned string and panics.
```
[WARN] - (starship::config): Failed to load config value: Error in 'OS' at 'symbols': invalid type: string "Alpine", expected a borrowed string
```
`HashMap` also has the same error. Though, switching to `HashMap` is likely better regardless of this issue since the index and order does not matter for the module.

Reply:
Interesting, the `ValueDeserializer` is supposed to forward a borrowed string. `IndexMap` is useful as it keeps the `print-config` ordering predictable, though the std `BTreeMap` type should also handle this.